mapboxcli/scripts/directions.py

```diff
@@ -7,15 +7,17 @@
 
 @click.command(short_help="Routing between waypoints.")
 @click.argument('waypoints', nargs=-1)
-@click.option('--profile', default="mapbox.driving", type=click.Choice([
-              'mapbox.driving', 'mapbox.walking', 'mapbox.cycling']),
-              help="Mapbox direction profile id")
+@click.option('--profile', default="mapbox.driving",
+              help="Mapbox direction profile id, must be one of {0}".format(
+                  mapbox.Directions().valid_profiles))
```

Comment:
The only reason I switched to help text was a) following the lead of geocoding and b) I don't like the way the enumerated list looks in the help text when using `Choice`: if you look at the current readme, see how they overlap into the right column? Maybe that's a silly reason - I happy to change it back to `click.Choice` if it makes more sense to you